### DISCUSSION TITLE
Automated Custom Status

### DISCUSSION THREAD

**Discussion by oumellouk** [stoatchat/discussions#83] [Feature Suggestions]
yall knows that the most chatting apps like ( Skype, WhatsApp , and also discord ) have  Custom Status but not at all have the same what users want or the style of it. 
so imagine an automated custom status with automated system made by you , yes seriously by you and u can control it 
like playing Revolt Desktop App will detect what are u playing ( Among US , Fortnite , Minecraft , Roblox .. ) and will set an Custom Status in app Like ** User000 is playing Fortnite ** 
about music like ( spotify , soundcloud , Apple Music ) ** User000 is listening on spotify **
about codding like ( sublime text , visual studio code , notepad++ ) ** User000 is Editing on VSC **

**Comment by gdude2002** [comment]
I don't fully understand what you're asking for - from my reading, this is what I'm getting:

* Revolt should be able to detect what apps you're using, and allow you to configure them to be shown in your user presence
* Revolt should allow you to configure the *verb* used to explain what you're doing in the app

If that's the case, it may be worth pointing out that allowing users to supply any verb isn't possible for translations - but I imagine a list of known verbs could work.

**Reply by oumellouk** [reply]
You get the idea and you understand what i mean and yes allowing users to supply any verb isn't possible for translations is very good
